cirq-core/cirq/contrib/qasm_import/_parser.py

```diff
@@ -47,6 +60,29 @@ def __init__(
         self.circuit = c
 
 
+def _generate_op_qubits(args: List[List[ops.Qid]], lineno: int) -> List[List[ops.Qid]]:
+    # OpenQASM gates can be applied on single qubits and qubit registers.
+    # We represent single qubits as registers of size 1.
+    # Based on the OpenQASM spec (https://arxiv.org/abs/1707.03429),
+    # single qubit arguments can be mixed with qubit registers.
+    # Given quantum registers of length reg_size and single qubits are both
+    # used as arguments, we generate reg_size GateOperations via iterating
+    # through each qubit of the registers 0 to n-1 and use the same one
+    # qubit from the "single-qubit registers" for each operation.
```

Comment:
Do you think this should be a triple quoted doc string for the private function rather than a 7-line line comment?